cmd/configure.go

```diff
@@ -589,6 +591,10 @@ func configureGithub(ctx context.Context, flags ConfigureGithubFlags) error {
 			secrets[key] = val
 		}
 
+		if mode, ok := generationWorkflow.Jobs.Generate.With[config.Mode].(string); ok && mode == "pr" {
```

Comment:
Not to get carried away here, but it would be really nice to add an `isPRMode()` method to the underlying model here so we can just call `generationWorkflow.isPRMode()`

Reply:
I do agree

Reply:
I'll make a mental note and maybe do today haha